llmware/library.py

```diff
@@ -673,7 +1022,24 @@ def update_block (self, doc_id, block_id, key, new_value):
 
     def add_website (self, url, get_links=True, max_links=5):
 
-        """ Main method to ingest a website into a library """
+        """ Main method to ingest a website into a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Main method to ingest a website into a library.
```